### PR TITLE
Kia: Car Port for 'Picanto 2022'

### PR DESCRIPTION
**Pull request by jun980401**
Checklist

- [ ] Registered vehicle information in 'values.py' (**Fingerprint value not obtained yet -> commented it out now**)
- [ ] Add vehicle specification to 'interface.py'

Any advice would be much appreciated.

### CODE REVIEW

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -66,6 +66,7 @@ class CAR:
   KIA_SORENTO = "KIA SORENTO GT LINE 2018"
   KIA_STINGER = "KIA STINGER GT2 2018"
   KIA_CEED = "KIA CEED INTRO ED 2019"
+  KIA_MORNING = "KIA ALL NEW MORNING 2019"
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/values.py:69]: this car seems to have many different regional names, I propose referring to it as the `Kia Picanto Morning` in openpilot. 

https://en.wikipedia.org/wiki/Kia_Picanto

Comment (jun980401) [reply selfdrive/car/hyundai/values.py:69]: Thank you so much.
I hadn't thought of that part. I'll fix it as you said.

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -221,6 +223,12 @@ class Buttons:
   CAR.PALISADE: [{
     67: 8, 127: 8, 304: 8, 320: 8, 339: 8, 356: 4, 544: 8, 546: 8, 547: 8, 548: 8, 549: 8, 576: 8, 593: 8, 608: 8, 688: 6, 809: 8, 832: 8, 854: 7, 870: 7, 871: 8, 872: 8, 897: 8, 902: 8, 903: 8, 905: 8, 909: 8, 913: 8, 916: 8, 1040: 8, 1042: 8, 1056: 8, 1057: 8, 1064: 8, 1078: 4, 1107: 5, 1123: 8, 1136: 8, 1151: 6, 1155: 8, 1156: 8, 1157: 4, 1162: 8, 1164: 8, 1168: 7, 1170: 8, 1173: 8, 1180: 8, 1186: 2, 1191: 2, 1193: 8, 1210: 8, 1225: 8, 1227: 8, 1265: 4, 1280: 8, 1287: 4, 1290: 8, 1292: 8, 1294: 8, 1312: 8, 1322: 8, 1342: 6, 1345: 8, 1348: 8, 1363: 8, 1369: 8, 1371: 8, 1378: 8, 1384: 8, 1407: 8, 1419: 8, 1427: 6, 1456: 4, 1470: 8, 1988: 8, 1996: 8, 2000: 8, 2004: 8, 2005: 8, 2008: 8, 2012: 8
   }],
+  """
+  Put KIA_MORNING Fingerprint below(not yet found).
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:227]: You can skip this (fpv1), just need to make sure you add the fpv2 (firmware versions) below.

Comment (jun980401) [reply selfdrive/car/hyundai/values.py:227]: Got it. I'll look for fpv2.
Thanks for the advice.

**Comment by sshane** [comment]
Do you have a dongle ID/route for this car?

**Comment by jun980401** [comment]
> Do you have a dongle ID/route for this car?

@sshane No, unfortunately nothing. Just want to contribute for my future car.

**Comment by adeebshihadeh** [comment]
Car ports require access to a car. Let's revisit this once you have the car.

**Pull request closed without merging**